homeassistant/components/remote/itach.py

```diff
@@ -0,0 +1,98 @@
+"""
+Control an itach ip2ir gateway using libitachip2ir.
+
+Requires a command file with Philips Pronto learned hex formats.
+
+Example command file:
+
+OFF
+0000 006d 0022 0002 0157 00ac 0015 0016 0015 0016 0015 0041 0015
+
+ON
+0000 006d 0022 0002 0157 00ac 0015 0016 0015 0016 0041 0015 0015
+
+Example configuration.yaml section:
+remote living_room:
+  - platform: itach
+    name: Living Room
+    mac: 000C1E023FDC
+    ip: itach023fdc
+    port: 4998
+    devices:
+      - { name: TV,  connaddr: 2, file: Samsung_TV.txt }
+      - { name: DVD, connaddr: 3, file: LG_BR.txt }
+"""
```

Comment:
Please format the comment like this:

```
"""
Support for iTach IR Devices.

For more details about this platform, please refer to the documentation at
https://home-assistant.io/components/remote.itach/
"""
```

Details like you provided belong in the documentation.